### ISSUE TITLE
Visibility of "private" tasks

### ISSUE THREAD

**Issue by raptwa** [weglide/bugtracker#116]
**Describe the bug**
When setting the visibility for a task within the planer to private, the task still can be found by the search.

**To Reproduce**
1. Go to task planer
2. Create task
3. Set visibility to private
4. Logout and login with different account
5. Search for the created task

**Expected behavior**
When setting the visibility to private, the task should not be found by the search.

**Device (please complete the following information):**
all

**Comment by samuel-git** [comment]
This is definitely a bug. @flyingjoe.

**Comment by flyingjoe** [comment]
I can't reproduce this issue and thought we fixed it about a month ago.
Could you try this again? @raptwa

**Comment by raptwa** [comment]
Yep, I too can't reproduce it. Seems to be fixed. Despite this, one week ago the behaviour was definitely as described in the inititial post not only for me.
